docs/_plugins/liquify.rb

```diff
@@ -0,0 +1,9 @@
+module Jekyll
```

Comment:
Missing license header. Also, can you add some docs on this filter. I realize it's trivial but we've had issues of no one understanding these parts of the docs build in the past.